bindings/java/src/main/java/org/openucx/jucx/examples/UcxReadBWBenchmarkReceiver.java

```diff
@@ -92,6 +92,14 @@ public void onSuccess(UcxRequest request) {
             worker.progress();
         }
 
+        // Close endpoint and wait for remote side
+        // TODO remove when UCP close protocol is implemented
```

Comment:
Maybe move this to EndpointTest ?

Reply:
you mean copy this to EndpointTest?
probably not needed, since EndpointTest is single process so we are sure both endpoints are disconnected before any worker is destroyed

Reply:
it's related to close protocol because with close protocol we could destroy the worker, and rdmacm (or tcp) driver would handle remote side disconnect